test/test_multispecies_problem.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import festim as F
+from petsc4py import PETSc
+from dolfinx.fem import Constant
+from ufl import exp
+import os
+
+
+def test_multispecies_permeation_problem(tmp_path):
```

Comment:
I have no idea what this test does, the name of the function isn't helping.

By reading it I see it checks against an analytical solution.
Analytical solution of what? Concentration field? Flux? if flux, which flux? left surface? right surface? left for D and right for T? the opposite?